src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -496,7 +492,7 @@ public static String getHostname(EnvVars envVars) {
         String hostname = null;
```

Comment:
I'd replace this catch with an explicit null-check:
```
DatadogGlobalConfiguration datadogConfiguration = getDatadogGlobalDescriptor();
if (datadogConfiguration != null) {
    hostname = datadogConfiguration.getHostname();
}
```

Reply:
Done.